core/iwasm/common/wasm_c_api.c

```diff
@@ -5202,3 +5198,16 @@ BASIC_FOUR_LIST(WASM_EXTERN_AS_OTHER_CONST)
 
 BASIC_FOUR_LIST(WASM_OTHER_AS_EXTERN_CONST)
 #undef WASM_OTHER_AS_EXTERN_CONST
+
+wasm_extern_t *
+wasm_extern_new_empty(wasm_store_t *store, wasm_externkind_t extern_kind)
+{
+    if (extern_kind == WASM_EXTERN_FUNC)
+        return wasm_func_as_extern(wasm_func_new_empty(store));
```

Comment:
Should we check the return value of `wasm_func_new_empty` since it may be NULL?

Reply:
`wasm_func_as_extern` will return NULL if the input is NULL.

https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/core/iwasm/common/wasm_c_api.c#L5178-L5181

Reply:
Got it